client/src/screens/events/Detail.js

```diff
@@ -362,7 +362,7 @@ class Detail extends Component {
                 title={event.name}
                 subtitle={event.details}
                 icon="external-link"
-                onPress={() => undefined}
+                onPress={() => { Linking.openURL(event.permalink); }}
```

Comment:
please move this to a component instance method so you can save a re-render every time it updates (which is at least every 5 seconds)
```
onPress={this.openExternal}
```